Helper/Vault.php

```diff
@@ -196,12 +201,12 @@ private function validateAdditionalData(array $additionalData)
      * @return string
      * @throws Exception
      */
-    private function getExpirationDate($expirationDate)
+    public function getExpirationDate($expirationDate)
```

Comment:
This is not meant to be public. Not here, at least.

Reply:
Switched it